salt/_states/caasp_file.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import absolute_import
+
+import salt.utils.files
+
+
+def managed(name, **kwargs):
+    '''
+    Manage a given file, this function allows for a file to be downloaded from
+    the salt master and potentially run through a templating system.
+
+    This is a wrapper on the standard :py:func:`file.managed <salt.states.file.managed>`
+    state where we can specify a `work_dir` that will be used for creating the temporary
+    file, as the standard version creates a temporary file in the same directory as `name`,
+    and that can lead to some problems with programs/daemons that are watching that
+    directory (like the kubelet with `/etc/kubernetes/manifests`).
+
+    work_dir
+        A directory for creating temporary files.
+
+    For a full list of arguments see :py:func:`file.managed <salt.states.file.managed>`
+    '''
+    def debug(s):
+        __utils__['caasp_log.debug']('CaaS: file.managed: {}: '.format(name) + s)
```

Comment:
`nit` should be `CaaSP: file.managed blabla` or `CaaS: blab`?

Reply:
Yes, this is wrong. I will fix it.